content/docs/rfcs/17/README.md

```diff
@@ -2,127 +2,138 @@
 slug: 17
 title: 17/WAKU-RLN-RELAY
 name: Waku v2 RLN Relay
-status: raw
+status: draft
 tags: waku-core
 editor: Sanaz Taheri <sanaz@status.im>
 ---
```

Comment:
```suggestion
editor: Sanaz Taheri <sanaz@status.im>
contributors: Oskar Thoren <oskar@status.im>
---
```

Gonna add myself to this one :D

Reply:
Sure, done https://github.com/vacp2p/rfc/pull/484/commits/8954039fe162421351bcacf73cba41ef864bca21